torch/_torch_docs.py

```diff
@@ -8461,7 +8461,48 @@ def merge_dicts(*dicts):
 """.format(**common_args))
 
 add_docstr(torch.std, r"""
-std(input, dim, unbiased, keepdim=False, *, out=None) -> Tensor
+std(input, dim=None, *, correction=1, keepdim=False, out=None) -> Tensor
+
+Calculates the standard deviation over the dimensions specified by :attr:`dim`.
+:attr:`dim` can be a single dimension, list of dimensions, or ``None`` to
+reduce over all dimensions.
+
+The standard deviation (:math:`\sigma`) is calculated as
+
+.. math:: \sigma = \sqrt{\frac{1}{N - \delta N}\sum_{i=0}^{N-1}(x_i-\bar{x})^2}
+
+where :math:`x` is the sample set of elements, :math:`\bar{x}` is the
+sample mean, :math:`N` is the number of samples and :math:`\delta N` is
+the :attr:`correction`.
+""" + r"""
+
+{keepdim_details}
+
+Args:
+    {input}
+    {dim}
+
+Keyword args:
+    correction (int): difference between the sample size and sample degrees of freedom.
+                      Defaults to Bessel's correction, ``correction = 1``.
```

Comment:
Is it worth mentioning that when correction = 1 this is an unbiased estimator of the standard deviation?

Reply:
I've added a link to the wiki for Bessel's correction. IMO It's a bit more complicated than just being unbiased, it would have to be more like 
> assuming the input is a sample from a normally-distributed population, this gives an unbiased estimator of the population standard deviation.

which is probably too wordy for an argument description.